src/index.html

```diff
@@ -7,6 +7,392 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img src="images/logo.png" alt="Moyo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__item" >
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a 
+              class="nav__link" 
+              data-qa="nav-hover" 
+              href="#"
+            >Ноутбуки и компьютеры</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="container">
+        <div class="container__card">
+          <div 
+          data-qa="card" 
+          class="card">
+            <div class="card__image">
+              <img 
+                src="./images/imac.jpeg" 
+                alt="Imac" 
+                class="card__img">
+            </div>
+            <div class="card__title">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </div>
```

Comment:
isn't it `h`?

Reply:
yes it is h. But I don't know which p to set, depending on the context of the page.
h3?